### PR TITLE
Landing page

### PR DESCRIPTION
**Pull request by bonniezhou**
Set up the catalog landing page with a (vanilla) top app bar. This PR includes:
- ejecting webpack!!
- installing `sass-loader` from npm and configuring it in the dev and prod webpack configs
- installing `@material/top-app-bar` and dependencies from npm
- using top app bar JSX
- importing top app bar CSS
- using the appropriate favicon

To see the demo, run `npm run start` and point your browser to http://localhost:3000/

### CODE REVIEW

File: config/webpack.config.dev.js [hunk 1]
```diff
@@ -0,0 +1,275 @@
+'use strict';
```
Comment (moog16) [inline config/webpack.config.dev.js]: I assume everything above this file was generated from `eject`

File: src/App.js [hunk 2]
```diff
@@ -4,7 +4,13 @@ import './App.scss';
 class App extends Component {
```
Comment (moog16) [inline src/App.js]: Besides the 2 dev/prod webpack config files, I believe this is the first file you created, yes?

File: src/App.js [hunk 3]
```diff
@@ -4,7 +4,13 @@ import './App.scss';
...
   render() {
     return (
-      <h1>Hello World!</h1>
+      <header className="mdc-top-app-bar">
```
Comment (moog16) [inline src/App.js]: Since we're in JS land, lets use single quotes. We should add this to the eslinter.

File: src/App.scss [hunk 4]
```diff
@@ -1,3 +1,5 @@
+@import "@material/top-app-bar/mdc-top-app-bar";
+
 .App {
```
Comment (moog16) [inline src/App.scss]: Also looks like there is margin around the `body`. Add:

```
body { margin: 0; }
```

File: config/webpack.config.dev.js (hunk 1 above)
Comment (bonniezhou) [reply config/webpack.config.dev.js]: Yep, the only files I manually changed were this one, `webpack.config.prod.js`, `index.html` (for the favicon), `App.js`,  and `App.scss`.

File: src/App.js (hunk 3 above)
Comment (bonniezhou) [reply src/App.js]: Done. This repo doesn't have an eslinter yet, I'll add that in another PR.

File: .eslintrc [hunk 5]
```diff
@@ -0,0 +1,7 @@
+{
+  extends: "react-app",
```
Comment (moog16) [inline .eslintrc]: Should probably use singles here too :|

**Pull request merged**